apps/judicial-system/backend/src/app/messages/notifications.ts

```diff
@@ -122,19 +122,27 @@ export const notifications = {
   defenderResubmittedToCourt: defineMessages({
     subject: {
       id:
-        'judicial.system.backend:notifications.defender_resubmitted_to_court.subject_v2',
-      defaultMessage: 'Krafa um {caseType} send aftur',
+        'judicial.system.backend:notifications.defender_resubmitted_to_court.subject',
```

Comment:
Does this not need to be `subject_v3`, based on what you were saying earlier about CF storing ids indefinitely?

Reply:
Yes